app/models/sitemap/robots_updatable.rb

```diff
@@ -0,0 +1,63 @@
+class Sitemap
+  module RobotsUpdatable
+    extend ActiveSupport::Concern
+
+    def handle_robots_update!
+      return unless update_robots
+
+      update_robots_file!
+
+      puts_robots_action
+    end
+
+    private
+
+    def update_robots_file!
+      File.write(robots_file_path, robots_content)
+    end
+
+    def puts_robots_action
```

Comment:
The method name `puts_robots_action` doesn't clearly indicate it's performing output. Consider renaming to `output_robots_action` or `log_robots_action` for clarity.